docker-compose.yml

```diff
@@ -8,8 +8,8 @@ services:
       context: .
       dockerfile: contrib/images/slinky.sidecar.prod.Dockerfile
     volumes:
-      - ./config/dydx/oracle.json:/oracle/oracle.json
-      - ./config/dydx/market.json:/oracle/market.json
+      - ./config/core/oracle.json:/oracle/oracle.json
+      - ./config/core/market.json:/oracle/market.json
```

Comment:
Is it possible to template this? I frequently have to edit this myself depending on what I'm testing